### PR TITLE
doc: Remove invalid statement about missing encryption in fastdp

### LINKED ISSUE #2830: Documentation for encryption needs update
The URL https://www.weave.works/docs/net/latest/using-weave/security-untrusted-networks/ contains a statement that is no longer true:

    Fast datapath does not support encryption. If you supply a password at weave launch, Weave Net falls back to the slower sleeve mode for encrypted communication.

This should be updated.

### PR DESCRIPTION
**Pull request by brb**
Fix #2830

### CODE REVIEW

**Comment by awh** [comment]
#### Documentation changes:
modified [using-weave/security-untrusted-networks](https://dev-weavewww.pantheonsite.io/docs/net/2830-fix-fastdp-crypto-docs/using-weave/security-untrusted-networks/)

**Comment by marccarre** [comment]
It is worth mentioning anything about Sleeve for older versions?

**Comment by bboreham** [comment]
I think you should fix "Weave Net attempts to establish fast datapath connectivity, which is unencrypted" too

While we're here, "the password needs to be reasonably strong with at least 50 bits of entropy is recommended" is wrong - either "the password needs to be reasonably strong, with at least 50 bits of entropy" or "the password needs to be reasonably strong - at least 50 bits of entropy is recommended"

**Comment by brb** [comment]
Damn, totally missed it. Thanks for noticing @bboreham .

**Comment by awh** [comment]
#### Documentation changes:
modified [using-weave/security-untrusted-networks](https://dev-weavewww.pantheonsite.io/docs/net/2830-fix-fastdp-crypto-docs/using-weave/security-untrusted-networks/)

File: site/using-weave/security-untrusted-networks.md [hunk 1]
```diff
@@ -15,23 +15,21 @@ or
...
+>**NOTE**: The command line option takes precedence over the environment variable.
 
 > To avoid leaking your password via the kernel process table or your
 > shell history, we recommend you store it in a file and capture it
 > in a shell variable prior to launching weave: `export
 > WEAVE_PASSWORD=$(cat /path/to/password-file)`
 
-To guard against dictionary attacks, the password needs to be reasonably strong with at least 50 bits of entropy is recommended. An easy way to generate a random password that satisfies this requirement is:
+To guard against dictionary attacks, the password needs to be reasonably strong,
+with at least 50 bits of entropy is recommended. An easy way to generate a random
```
Comment (bboreham) [inline site/using-weave/security-untrusted-networks.md]: this section seems to have been reformatted but still has poor grammar

**Comment by awh** [comment]
#### Documentation changes:
modified [using-weave/security-untrusted-networks](https://dev-weavewww.pantheonsite.io/docs/net/2830-fix-fastdp-crypto-docs/using-weave/security-untrusted-networks/)

**Pull request merged**